src/coreclr/vm/codeman.h

```diff
@@ -1912,6 +1916,16 @@ protected :
         return m_CPUCompileFlags;
     }
 
+#if defined(TARGET_ARM64) && !defined(_MSC_VER)
```

Comment:
no need of this anymore?

Reply:
We need an "sve" attribute for GNUC builds. Not sure if `#if defined(TARGET_ARM64) && defined(__GNUC__)` is enough here.

Reply:
I can only test on Arm64 host with Linux base systems at the moment so have to spam this CI for other build combinations.

Reply:
You do not need this definition because you will be using the one that is defined in asmhelpers.* files.

Reply:
Apologies for the delay in responding. It doesn't compile after removing the definition of `GetSveLengthFromOS` from here.
The issue here is that on Linux, if we rely on asmhelpers.* files, we get compilation error `error: instruction requires: sve or sme`. 
We avoid this error by adding the `__attribute__((target("sve")))`. However, this is attribute not defined for MSVC so we use asmhelpers.* files to define `GetSveLengthFromOS` there.

Reply:
What does `__attribute__((target("sve")))` traslate to in the .S file? I think you should be able to make the .S file compile by adding assembly equivalent of `__attribute__((target("sve")))` to it.

Reply:
Have you looked into using  `.arch_extension sve` in the .S file to get the SVE instruction to compile? It would look better if the .asm and .S files are symmetric.